### DISCUSSION TITLE
Plotting phonon branch resolved eletron scattering rate.

### DISCUSSION THREAD

**Discussion by jk262626** [nakib/elphbolt#151] [Q&A]
Dear Nakib,

It looks like I could plot phonon scattering rate depending on the phonon branches. 
Would it be possible to export electron scattering rate interacting with each phonon branch from the elphbolt?

**Comment by nakib** [comment]
Dear Jiyoung,

Yes, it is in principle possible to plot the electron scattering rates for each phonon branch. But for that you will need to modify `src/interactions.f90 -> calculate_eph_interaction_ibzk`. You will find that there is a phonon bands loop:

```
!Run over phonon branches
do s = 1, wann%numbranches
...
```

You have to set your target branch there and compute the scattering rate one phonon band at a time. There is no cleaner way to do this at the moment.

Best,
Nakib
